apps/webapp/src/app/(Magic-Link)/magic-link/page.tsx

```
@@ -0,0 +1,614 @@
+          </div>
+        </DialogContent>
+      </Dialog>
+
+      {/* Success Dialog */}
+      <Modal
+        open={openSuccessDialog}
+        setOpen={CloseSuccessDialog}
+        backgroundClass="bg-black/50"
+      >
+        <div className="bg-white text-black rounded-lg shadow-lg max-w-md w-full p-0  overflow-hidden">
+          <div className="flex justify-between items-center p-4"></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)